src/xdocs/config_javadoc.xml

```diff
@@ -1213,6 +1213,158 @@ public int checkReturnTag(final int aTagIndex,
       </subsection>
     </section>
 
+    <section name="JavadocSummaryLocation">
+      <p>Since Checkstyle 8.26</p>
+      <subsection name="Description" id="JavadocSummaryLocation_Description">
+        <p>
+          Checks that <a
+          href="https://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#firstsentence">
+          Javadoc summary sentence</a> is in the same position for all javadoc
+          comments in the project.
+        </p>
+        <p>
+          It is possible to enforce two different styles:
+        </p>
+        <ul>
+          <li>
+            {@code First line} - javadoc summary starts on the first line:
+            <source>
+&#47;** Summary text.
+* More details.
+*&#47;
+public void method();
+            </source>
+          </li>
+          <li>
+            {@code Second line} - javadoc summary starts on the second line:
+            <source>
+&#47;**
+* Summary text.
+* More details.
+*&#47;
+public void method();
+            </source>
+          </li>
+        </ul>
+      </subsection>
+
+      <subsection name="Properties" id="JavadocSummaryLocation_Properties">
+        <table>
+          <tr>
+            <th>name</th>
+            <th>description</th>
+            <th>type</th>
+            <th>default value</th>
+            <th>since</th>
+          </tr>
+          <tr>
+            <td>violateExecutionOnNonTightHtml</td>
+            <td>
+              Control when to print violations if the Javadoc being examined by this check
+              violates the tight html rules defined at
+              <a href="writingjavadocchecks.html#Tight-HTML_rules">Tight-HTML Rules</a>.</td>
+            <td><a href="property_types.html#boolean">Boolean</a></td>
+            <td><code>false</code></td>
+            <td>8.26</td>
+          </tr>
+          <tr>
+            <td>location</td>
+            <td>Specify the policy on placement of the javadoc summary.</td>
+            <td>
+              <a href="property_types.html#javadocSummaryLocation">Javadoc Summary Location</a>
+            </td>
+            <td><code>second_line</code></td>
+            <td>8.26</td>
+          </tr>
+        </table>
+      </subsection>
+
+      <subsection name="Examples" id="JavadocSummaryLocation_Examples">
+        <p>
+          By default Check validate that the summary is located on the second line:
+        </p>
+        <source>
+&lt;module name=&quot;JavadocSummaryLocationCheck&quot;/&gt;
+        </source>
+        <p>
+          The following code produces a violation because a blank line is
+          required before the summary:
+        </p>
+        <source>
+&#47;** This comment starts on the first line
+  * and spans several lines.
+  *&#47;
```

Comment:
Please do examples like:
```
/** This comment starts on the first line (it is violation)
  * and spans several lines.
  */
/** 
  * This comment starts on the second line (it is valid)
  * and spans several lines.
  */
```
user need to see config and what code is valid and invalid. Terminology "first line" and "seconds line" is not certain, invented by us, so user need to see what does it mean.

Reply:
done